### DISCUSSION TITLE
Table Column hidden no longer being honoured - potential new bug?

### DISCUSSION THREAD

**Discussion by Blackpig** [filamentphp/filament#216] [General]
Hey,

Just jumping back onto using this and my workaround for having a searchable attribute column (https://github.com/laravel-filament/filament/discussions/51) is no longer viable because the columns I want to search are no longer honouring the hidden() setting.  So I have my full_name column populated from the model getFullNameAttribute but now my 2 hidden fields are also displaying.

```
return $table
            ->columns([
                Columns\Text::make('full_name')
                    ->primary(),
                Columns\Text::make('first_name')
                    ->searchable()
                    ->hidden(),
                Columns\Text::make('last_name')
                    ->searchable()
                    ->hidden(),
            ])
```

I don't know when the change was made as I have just upgraded to v1.6.1 

Cheers

**Comment by danharrin** [comment]
Hey! This should be fixed with v1.6.2.
